app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -277,7 +281,15 @@ def find_task
       end
 
       def resource_scope(_options = {})
-        @resource_scope ||= ForemanTasks::Task.authorized("#{action_permission}_foreman_tasks")
+        @resource_scope ||= begin
```

Comment:
Just so we're not building it over and over again, but I can drop it

Reply:
Well I think it should get used once per action, so there shouldn't be the need for it and it would tidy up the code, but I'm not sure if there was some reason in the first place.